### PR TITLE
Add abstracts test

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- add test abstracts oct 18

### CODE REVIEW

**Review by ewels — APPROVED** [review]
It'd be nice to highlight the keynote talks, as we did on the speaker list.

Also to add `is_sponsor` with similar highlighting on talks + speakers page. But those are both bonuses. It's good to merge whenever you're ready I think 👍🏻 

See Slack DM for detailed spec of what I'm thinking with the new styling ☝🏻

File: gatsby-node.js [hunk 1]
```diff
@@ -39,6 +39,8 @@ exports.createSchemaCustomization = ({ actions }) => {
         slug: String
         path: String
         fullPath: String
+				is_keynote: Boolean
+				is_sponsor: Boolean
```
Comment (ewels) [inline gatsby-node.js:43]: Tabs?? 😱 😆 

```suggestion
        is_keynote: Boolean
        is_sponsor: Boolean
```

Comment (MythicalFish) [reply gatsby-node.js]: The horror 😱 (fixed lol)

Comment (ewels) [reply gatsby-node.js]: And prettier too 😅

**Pull request merged**